pontoon/base/static/js/translate.js

```diff
@@ -2723,6 +2711,23 @@ var Pontoon = (function (my) {
     attachMainHandlers: function () {
       var self = this;
 
+      // Main keyboard shortcuts
+      $('html').on('keydown', function (e) {
+        var key = e.which;
```

Comment:
I wonder if something like this:
```
var direction = {
 40: 'next',
 38: 'previous'
}[e.which]; //yeah, that can be moved into something nicer
if (e.altKey && direction) {
   self.navigateToEntity(direction);
   return false;
}
```
Would reduce complexity/number of nested blocks? what do you think about that approach?